staff/gianluca-bragaglia/cv/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My CV</title>
+</head>
+<body>
+    <header>
+        <h1>John Dor</h1>
+        <h2>Web Developer</h2>
+        <address>
+            151 East 2º street <br> 
+            New York 12345 <br>
+            <a href="mailto:info@dor.com">info@dor.com</a>
+        </address>
+        <hr>
+    </header>
+    <br>
+    <section>
+        <h2>Goals</h2>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nisi sunt est suscipit odio repellat at fugit aspernatur. Unde, impedit quo libero, nemo eligendi, at doloribus doloremque aut inventore perspiciatis blanditiis.</p>
+    </section>
+    <br>
+    <br>
+    <section>
+        <h2>Skills</h2>
+        <article>
```

Comment:
article does not seem to fit here, as the content you are describing doesn't have the kind of an article, but of a section. please, RTFM: 
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
changed